src/board.py

```diff
@@ -1,25 +1,10 @@
 class Board:
     SIZE = 9
-    border = "⊱ –––––– {⋆⌘⋆} –––––– ⊰",
-    divider = "      ---+---+---"
     space = " "
```

Comment:
Seems like a blank space is also a visual concern. Have you considered making null the representation of blank space at the board level, and blank space being the visual representation of a null space?

Reply:
That's a really interesting point... Let me think a little bit about how to tackle that